tests/queries/0_stateless/02381_rocksdb_read_only.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS 02381_test;
```

Comment:
```suggestion
-- Tags: no-fasttest

DROP TABLE IF EXISTS 02381_test;
```

CH isn't built with RocksDB support for fast tests so that's why it fails.
Sorry, should've caught that earlier.

Reply:
Get, thanks a lot.